docs/topics/Permissions.md

```diff
@@ -15,8 +15,9 @@ Permissions in Discord are stored within a 53-bit integer and are calculated usi
 | MANAGE\_CHANNELS * | `0x00000010` | Allows management and editing of channels |
 | MANAGE\_GUILD * | `0x00000020` | Allows management and editing of the guild |
 | ADD\_REACTIONS | `0x00000040` | Allows for the addition of reactions to messages |
+| VIEW\_AUDIT\_LOG | `0x00000080` | Allows for viewing of audit logs. |
```

Comment:
Should be pluralised similar to the UI (and the actual section in the guild settings).